static/scss/videoPlayer.scss

```diff
@@ -28,6 +26,11 @@
   position: relative !important;
   width: inherit !important;
   height: inherit !important;
+  max-height: calc(100vh - 200px);
+
+  @include breakpoint(phone) {
+    max-height: calc(100vh - 60px);
```

Comment:
This doesn't quite seem to fix it for me - I think what we need to do is apply the `max-height` only when the video player is not fullscreen. I think something like this would do it:

```scss
.video-js {
  &:not(.vjs-fullscreen) {
    .vjs-tech {
      max-height: calc(100vh - 200px);
      
      @include breakpoint(phone) {
        max-height: calc(100vh - 60px);
      }
    }
  }
}
```

so that we don't set a `max-height` when the thing is fullscreened, but we do otherwise.